src/examples/zkapps/reducer/run-live.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  AccountUpdate,
+  Lightnet,
+  Mina,
+  PrivateKey,
+} from '../../../../dist/node/index.js';
+import { randomAccounts } from '../../utils/randomAccounts.js';
+import { tic, toc } from '../../utils/tic-toc.node.js';
+import { MerkleListReducing } from './actions-as-merkle-list.js';
+
+const useCustomLocalNetwork = process.env.USE_CUSTOM_LOCAL_NETWORK === 'true';
+
+if (!useCustomLocalNetwork) {
+  throw 'Only Lightnet is currently supported';
+}
+
+tic('Run reducer examples against real network.');
+console.log();
+const network = Mina.Network({
+  mina: useCustomLocalNetwork
+    ? 'http://localhost:8080/graphql'
+    : 'https://berkeley.minascan.io/graphql',
+  archive: useCustomLocalNetwork
+    ? 'http://localhost:8282'
+    : 'https://api.minascan.io/archive/berkeley/v1/graphql',
+  lightnetAccountManager: 'http://localhost:8181',
+});
+Mina.setActiveInstance(network);
+
+let { keys, addresses } = randomAccounts('contract', 'user1', 'user2');
+
+let pendingTx: Mina.PendingTransaction;
+
+// compile contracts & wait for fee payer to be funded
+const senderKey = useCustomLocalNetwork
```

Comment:
same as above, I think this would simplify and improve readability by a ton (until we run it with a live public network)